### ISSUE TITLE
Genome browser

### ISSUE THREAD

**Issue by jimdavis1** [PATRIC3/patric3_website#1213]
I keep trying to look at the genes on the ends of contigs and I cant click or mouse over the left-most protein because it is blocked by that stupid "PATRIC Annotation" thingy.

![image](https://cloud.githubusercontent.com/assets/7661533/19969753/ba14d8b8-a1a7-11e6-9fed-701000e63857.png)

**Comment by dmachi** [comment]
This is part of JBrowse, so we'll have to see what we can do to avoid it.  If we just remove it altogether then there is no way to tell which track is which (though it may be obvious to someone like Jim if you are zoomed in far enough to see the feature labels).  

I found this plugin, https://github.com/GMOD/jbrowse/tree/master/plugins/HideTrackLabels  whether other people complained about the same thing.  I am not sure how to add this plugin yet or if it does something good, but it is worth looking into.

**Comment by mshukla1** [comment]
I agree with Dustin. This issue has always been there. 

Given that you can see the partial feature with adequate zooming and mouseover the feature to get more information, I consider this not critical, but nice to have feature.

**Comment by aswarren** [comment]
Added through plugin. Hidey button in navbox
